sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### Other Changes
 
+- Removed `_internal` APIs from the public API surface. Also removed most of the `_internal` APIs from the public `tracing.hpp` header.
+
+
```

Comment:
Please have only one blank line between version number and the last sentence here.